static/panes/editor.js

```diff
@@ -267,20 +267,26 @@ Editor.prototype.initCallbacks = function () {
 };
 
 Editor.prototype.initButtons = function (state) {
+    this.location = "PARENT"
```

Comment:
What is this used for? (On mobile currently and could not find where it is used)